services/wireguard/endpoint/netstack-provider/private.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2022 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package netstack_provider
+
+import (
+	"net"
+)
+
+func mustParseCIDR(cidrs []string) []*net.IPNet {
+	ipnets := make([]*net.IPNet, len(cidrs))
+	for i, cidr := range cidrs {
+		_, ipnet, err := net.ParseCIDR(cidr)
+		if err != nil {
+			panic(err)
+		}
+		ipnets[i] = ipnet
+	}
+	return ipnets
+}
+
+var privateIPv4Block []*net.IPNet
+
+var txtPrivateIPv4Block = []string{
```

Comment:
Please use the already existing flag `firewall.protected.networks` for this instead of hardcoded values. Some might want to adjust it. 
https://github.com/mysteriumnetwork/node/blob/master/config/flags_node.go#L96-L101

Reply:
Without allowing 10.0.0.0/8 web sites can not be accessed.
So it has to be overriden ?

Reply:
You need to allow only single IP like `10.182.0.2` which is coming from the session, the rest of the traffic should be blocked.

Reply:
yes, access to 10.182.0.1 is reqired